mmdet/datasets/pipelines/transforms.py

```diff
@@ -2789,19 +2789,22 @@ class CopyPaste:
             all objects of the source image will be pasted to the
             destination image.
             Default: True.
+        using_cutmix (bool): Whether using cutmix or not. BTW, cutmix in
```

Comment:
Why do we need this parameter?

Reply:
With the insight from @Czm369, the copypaste based on bbox is equal to the cutmix in object detection task. So, we rename the original param "mask_gen", which controls the copypaste wheather based on bbox or not, to "using_cutmix".

Reply:
using_cutmix is a bit vague and not clear to users. Suggest simply using `paste_by_box`, then users will know originally it is pasted by mask, and when it is on, this paste is done by box.

Reply:
Thanks for your suggestion. I think `paste_by_box` is a better choice.

Reply:
I just reviewed the logic of the code and found @hhaAndroid is right. We don't need this parameter!

The function `get_gt_masks` will help us to detect whether we have 'mask' annotations or not. If not, `gt_masks` will be automatically generated by `gen_masks_from_bboxes`.  And, It's more reasonable to `pop` the key `gt_masks` in the final `results` . So, an indicator is needed to point out whether the `gt_masks` is artificial or not. In the original version, `mask_gen` was playing such indicator role. 

In my opinion, we don't need any parameter, whatever `using_cutmix` or `paste_by_box`. @Czm369 @ZwwWayne

Reply:
Agree, we can do that.